common/metrics/tags.go

```diff
@@ -30,6 +30,7 @@ import (
 	"strings"
 
 	enumspb "go.temporal.io/api/enums/v1"
+	serverenumbspb "go.temporal.io/server/api/enums/v1"
```

Comment:
the convention in the server code base is `enumsspb`. I'll just fix all these in #4261

Reply:
We already have `enumspb` in the line above this.

Reply:
yeah, that's why it's `enums_s_pb` (don't shoot the messenger)

(actually I find it totally fine, they have totally disjoint contents, and when you're reading you don't really care which one a constant is from)